samples/nrf_desktop/src/events/config_event.h

```diff
@@ -20,33 +20,71 @@
 extern "C" {
 #endif
 
-#define DATA_SIZE_USER_CONFIG 4
 
-#if (REPORT_SIZE_USER_CONFIG <= DATA_SIZE_USER_CONFIG)
-#error "Invalid size of report"
-#endif
+#define MAX_CONFIG_DYNDATA_SIZE 32
+
+/* Config event ID macros */
+#define GROUP_FIELD_POS		6
+#define GROUP_FIELD_SIZE	2
+#define GROUP_FIELD_MASK	BIT_MASK(GROUP_FIELD_SIZE)
+#define GROUP_FIELD_SET(group)	((group & GROUP_FIELD_MASK) << GROUP_FIELD_POS)
+#define GROUP_FIELD_GET(event_id) ((event_id >> GROUP_FIELD_POS) & GROUP_FIELD_MASK)
+
+#define TYPE_FIELD_POS		0
+#define TYPE_FIELD_SIZE		6
+#define TYPE_FIELD_MASK		BIT_MASK(TYPE_FIELD_SIZE)
+#define TYPE_FIELD_SET(type)	((type & TYPE_FIELD_MASK) << TYPE_FIELD_POS)
+#define TYPE_FIELD_GET(event_id) ((event_id >> TYPE_FIELD_POS) & TYPE_FIELD_MASK)
+
+#define CONFIG_EVENT_ID(group, type) ((u8_t)(GROUP_FIELD(group) | \
```

Comment:
GROUP_FIELD and TYPE_FIELD are undefined

Reply:
Fixed